modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/CircleMBLayer.java

```diff
@@ -285,7 +285,7 @@ public Color getCircleStrokeColor() throws MBFormatException {
      * Access circle-stroke-color as literal or function expression, defaults to black.
      *
      * @return The color of the circle stroke.
-     * @throws MBFormatException
+     * @throws MBFormatException JSON provided does conform to specification
```

Comment:
"does not" I suppose?

Reply:
changed to `JSON provided inconsistent with specificaiton`